### DISCUSSION TITLE
Timezone issue in 'Collector > Schedule' tab

### DISCUSSION THREAD

**Discussion by gikang82** [cloudforet-io/community#21] [General]
Hi~

I tried to change change timezone in 'Asset Inventory > Collector > Schedule' Tab

But they did not changed.
I wanna know why are this happens.
Is this intended or bugs?

![스크린샷 2022-11-18 오전 11 43 56](https://user-images.githubusercontent.com/53893313/202605018-e31440fc-e472-4d1a-be3d-013a2fbb069d.png)

**Comment by whdalsrnt** [comment]
Hi, @gikang82 

This issue is not a bug, it is intended.

Currently, Cloudforet's time zone policy is as follows.
- All timezones in backend services are stored in UTC.
- Changing the time based on the user's time zone only on the console.
- The time zone setting above shows the user's time zone setting as it is.

The above time zone change is temporarily reflected and the settings are not saved.
However, users can get it wrong, so I'll review it again.

**Comment by whdalsrnt** [comment]
This item is linked in the issue below.
- https://github.com/cloudforet-io/community/issues/22
